metric/system/process/process.go

```diff
@@ -222,7 +222,6 @@ func (procStats *Stats) pidFill(pid int, filter bool) (ProcState, bool, error) {
 	if procStats.skipExtended {
 		return status, true, nil
 	}
-	status = procStats.cacheCmdLine(status)
```

Comment:
I'm wondering if that should have stayed here or somewhere after setting `status.Cmdline`. I'll look into it and update this thread.

Reply:
Fixes by moving it back where it was and adding some comments.